runtime/src/lib.rs

```diff
@@ -836,6 +836,10 @@ impl Default for Call {
 
 parameter_types! {
     pub const MaxWhiteListSize: u32 = 20;
+    pub const ProposalsPostDeposit: Balance = 2000;
+    // module : proposals_discussion
+    pub const ProposalsDiscussionModuleId: ModuleId = ModuleId(*b"mo:propo");
```

Comment:
Please, change the module id to reflect the name of the proposals discussion module, not just proposals.